src/utils/proxy.js

```diff
@@ -386,6 +402,22 @@ const initializeProxy = async function ({ configPath, distDir, port, projectDir
         res.setHeader(key, val)
       })
 
+      const isUncaughtError = responseStatus === 500 && proxyRes.headers['x-nf-uncaught-error'] === '1'
```

Comment:
Do we need to check for the status code? I would think that the presence of the header alone should indicate an uncaught error.

Reply:
I do think it's a good check either way, if only from the perspective of readability, as we do set the header ourselves and a 500 is a clear message to the reader that we're dealing with an error.

Reply:
My concern is that the check is not broad enough. If the bootstrap starts producing an uncaught error with a different status code, the CLI will need to adjust to that.

Reply:
O yeah that makes sense :) In that case I will actually change it